bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -1305,7 +1310,7 @@ bool BCStateTran::onMessage(const FetchResPagesMsg *m, uint32_t msgLen, uint16_t
 
   uint32_t vblockSize = getSizeOfVirtualBlock(vblock, config_.sizeOfReservedPage);
 
-  Assert(vblockSize > sizeof(HeaderOfVirtualBlock));
+  AssertGE(vblockSize, sizeof(HeaderOfVirtualBlock));
```

Comment:
Is the change here intentional? (GE means >=)

Reply:
yes